File: CodeWars/Lucky_Number/sol.py
N = int(input())
for i in range(N):
   x = input()
   chk = 0
   for c in x:
      if c == '7':
         chk = 1
         break
   if chk:
      print(x)
      exit(0)

print("DOES NOT EXIST")

File: CodeWars/Lucky_Number/sol.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
   int n;
   cin >> n;
   vector<int> a(n);
   for (int &i : a) {
      cin >> i;
   }
   for (int i : a) {
      string s = to_string(i);
      if (count(s.begin(), s.end(), '7')) {
         cout << i ;
         return 0;
      }
   }
   cout << "DOES NOT EXIST";
}